rvic/core/history.py

```diff
@@ -646,27 +671,27 @@ def __write_array(self):
         outlet_decomp_ind[:] = self._outlet_decomp_ind
         onm[:, :] = char_names
 
-        for key, val in share.outlet_lon.__dict__.iteritems():
+        for key, val in iteritems(share.outlet_lon.__dict__):
```

Comment:
I don't think we need the `__dict__` portion here. Same for all instances below.